src/sentry/api/endpoints/project_trace_item_details.py

```diff
@@ -15,15 +17,35 @@
 from sentry.api.bases.project import ProjectEndpoint
 from sentry.api.exceptions import BadRequest
 from sentry.models.project import Project
+from sentry.search.eap.types import SupportedTraceItemType
+from sentry.search.eap.utils import translate_internal_to_public_alias
 from sentry.snuba.referrer import Referrer
 from sentry.utils import snuba_rpc
 
 
-def convert_rpc_attribute_to_json(source: dict) -> dict:
-    for k, v in source.items():
-        if k.startswith("val"):
-            return {"type": k[3:].lower(), "value": v}
-    raise BadRequest(f"unknown field in protobuf: {source}")
```

Comment:
There's no longer a fall through in the new code, is this intentional? Is it even possible that the RPC doesn't have an attribute that looks like `val*`?